cfme/tests/infrastructure/test_host.py

```diff
@@ -617,3 +617,22 @@ def test_infrastructure_hosts_crud(appliance, setup_provider):
     # Case6 - lastly do the delete. First try is canceled.
     host.delete(cancel=True)
     host.delete
+
+
+@test_requirements.infra_hosts
+def test_infrastructure_hosts_tagging(appliance, setup_provider):
+    """
+    Polarion:
+        assignee: prichard
+        casecomponent: Infra
+        caseimportance: high
+        initialEstimate: 1/6h
+    """
+    host = appliance.collections.hosts.all()[0]
```

Comment:
It may raise `IndexError` if not single host available for testing.
Its rare case but we can handle this and try to skip test if possible.

Reply:
I believe we have been assuming that there will be at lest one host. There always is with the providers we use for host testing. I created a fixture previously that checked for min hosts when we needed more than one. But it hasn't been used to check for just one. There are many tests in test_host.py that do this. If we really want to check I can add one, but it's not been the convention.

Reply:
Yup, I agreed that's why added rare case scenario like removal of host.